test/v3/addLiquidity/addLiquidity.liquiditybootstrapping.integration.test.ts

```diff
@@ -0,0 +1,281 @@
+// pnpm test -- test/v3/addLiquidity/addLiquidity.liquiditybootstrapping.integration.test.ts
+import {
+    Address,
+    createTestClient,
+    http,
+    publicActions,
+    TestActions,
+    walletActions,
+    parseUnits,
+    parseEther,
+    Hex,
+} from 'viem';
+
+import {
+    AddLiquidityKind,
+    Slippage,
+    PoolState,
+    CHAINS,
+    ChainId,
+    AddLiquidityInput,
+    PoolType,
+    Permit2Helper,
+    PERMIT2,
+    PublicWalletClient,
+    LBPParams,
+    CreatePoolLiquidityBootstrappingInput,
+    InitPool,
+    InitPoolInput,
+    balancerV3Contracts,
+} from '@/index';
+
+import { doCreatePool, getBalances } from '../../lib/utils/';
+
+import {
+    TOKENS,
+    setTokenBalances,
+    approveSpenderOnTokens,
+    sendTransactionGetBalances,
+    assertInitPool,
+} from '../../lib/utils';
+
+import { ANVIL_NETWORKS, startFork } from '../../anvil/anvil-global-setup';
+import { AddLiquidityV3 } from '../../../src/entities/addLiquidity/addLiquidityV3/index';
+
+const protocolVersion = 3;
+const poolType = PoolType.LiquidityBootstrapping;
+const chainId = ChainId.SEPOLIA;
+
+const BAL = TOKENS[chainId].BAL;
+const WETH = TOKENS[chainId].WETH;
+
+describe('add liquidity bootstrapping test', () => {
+    let client: PublicWalletClient & TestActions;
+    let tokens: Address[];
+    let rpcUrl: string;
+    let lbpParams: LBPParams;
+    let createPoolInput: CreatePoolLiquidityBootstrappingInput;
+    let poolAddress: Address;
+    let testAddress: Address;
+
+    beforeAll(async () => {
+        ({ rpcUrl } = await startFork(
+            ANVIL_NETWORKS[ChainId[chainId]],
+            undefined,
+            7831624n,
```

Comment:
Do you really need a separate blockNumber for this test? 
Ideally you should be able to update the global blockNumber so all tests run in the same block and we benefit from fork cache speeding up tests.

Reply:
I don't need them. I initially planned to add liq to an already deployed lbp. The test now however creates a lbp instead of using an existing one.